client/src/components/HomePageBody/CardPostList/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC } from "react";
+import { Post } from "../../../types/types";
+import Card from "./Card";
+
+import style from "./style.module.scss";
+
+const CardPostList: FC<{ posts: Post[] }> = ({ posts }) => {
+  return (
+    <div className={style.cardPostListContainer}>
+      {posts.map((post) => (
+        <Card post={post} />
```

Comment:
Nice! it's not really a Card component, right? it's your own custom card (great, but maybe just use a different naming?)